tests/Type/ArrayType/ArrayTypeTest.php

```diff
@@ -558,4 +558,17 @@ public function testUniqueValuesByCallbackWithNonStrictComparisonWithObjects()
 		$this->assertCount(1, $actual);
 	}
 
+	public function testHasKeyIsNotStrict()
```

Comment:
Please add a test for a null value - becase it differs from `isset` for example https://3v4l.org/Kd0c6
And any other weird values you know of :)